pkg/http/api_healthz.go

```diff
@@ -16,26 +16,43 @@ package http
 import (
 	"net/http"
 
+	"github.com/dapr/dapr/pkg/http/endpoints"
 	"github.com/dapr/dapr/pkg/messages"
 )
 
-func (a *api) constructHealthzEndpoints() []Endpoint {
-	return []Endpoint{
+var endpointGroupHealthzV1 = &endpoints.EndpointGroup{
+	Name:                 endpoints.EndpointGroupHealth,
+	Version:              endpoints.EndpointGroupVersion1,
+	AppendSpanAttributes: nil, // TODO
```

Comment:
All of these `TODO`'s are because these endpoints did not have anything defined in their configuration for tracing in the old methods in `pkg/diagnostics`.

Completing these TODOs should be done in separate PRs.